maps/away/smugglers/smugglers_items.dm

```diff
@@ -0,0 +1,95 @@
+/obj/item/paper/smug_1
+	name = "suspicios note"
```

Comment:
```suggestion
	name = "suspicious note"
```

Reply:
lmao didn't even notice that. Just copy and pasted it over